deploy/template.yaml

```diff
@@ -1345,7 +1345,7 @@ Resources:
                 - 'kms:Decrypt'
                 - 'kms:GenerateDataKey'
               Resource:
-                - !ImportValue CoreEncryptionKeyArn
+                - !Sub "arn:aws:kms:${AWS::Region}:${AWS::AccountId}:key/*"
```

Comment:
Is there any prefix we can add to restrict it further rather than allowing any key in our account?